standalone-metastore/metastore-tools/tools-common/src/main/java/org/apache/hadoop/hive/metastore/tools/Util.java

```diff
@@ -648,6 +682,15 @@ static void addManyPartitionsNoException(@NotNull HMSClient client,
             addManyPartitions(client, dbName, tableName, parameters, arguments, npartitions));
   }
 
+  static void addManyPartitionsNoException(@NotNull HMSClient client,
+                                           @NotNull String dbName,
+                                           @NotNull String tableName,
+                                           @Nullable Map<String, String> parameters,
+                                           List<List<String>> values) {
+    throwingSupplierWrapper(() ->
+            addManyPartitions(client, dbName, tableName, parameters, values));
```

Comment:
why not merge these two methods? `addManyPartitionsNoException` and `addManyPartitions`?

Reply:
Done.